interface/controller/pokemon_controller.go

```diff
@@ -0,0 +1,99 @@
+package controller
+
+import (
+	"net/http"
+	//"fmt"
+	//"errors"
+	"github.com/gorilla/mux"
+	"encoding/json"
+	"WizelineApi/domain/model"
+	"WizelineApi/service"
+	"strconv"
+)
+
+//Structura vacia del controlador. 
+type pokemonController struct{}
+
+type PokemonController interface{
+	GetAllPokemon()
+	GetPokemon()
+}
+
+
+
+func (pc *pokemonController) GetAllPokemons(res http.ResponseWriter, req *http.Request)  {
+	
+	pokemons := []model.Pokemon{}
+	err := service.ReadCSV("./test/bateria_csv/pokemon.csv", &pokemons)
+
+	if err != nil {
+		res.Header().Set("Content-Type", "application/json")
+		res.WriteHeader(http.StatusBadRequest)
+		res.Write([]byte(err.Error()))
```

Comment:
Write method returns an error. Propper handle them here and in other occurrences.

So the same for Encode() method.

Reply:
What I am doing this is to "render" an error. To the client (or response of HTTP CALL), what do you mean to return an "error"? 

How am I going to handle that return value of Error in the Router Handler?

Reply:
`res.Write()` returns two values `(int, error)`. You should never ignore errors returned by other methods.

What if an error occurs when calling the Write method? You are not going to render err.Error(), and you should do something to handle it.

Reply:
Oh thank you, on the documentation they didn't talk about the return value of calling that method, that's odd. But I will handle it, thanks